core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -49,18 +50,21 @@ import scala.util.control.NoStackTrace
   * Cluster mode can be set using the spark configuration
   * property spark.ext.h2o.mode which can be set in script starting sparkling-water or
   * can be set in H2O configuration class H2OConf
- */
+  */
 
 /**
   * Create new H2OContext based on provided H2O configuration
   *
   * @param sparkContext Spark Context
   * @param conf H2O configuration
   */
-class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf) extends org.apache.spark.Logging
+class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf, sqlc: SQLContext) extends org.apache.spark.Logging
```

Comment:
I have no objections. Removing this parameter.

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 12:21 AM, Jakub Háva <notifications@github.com>
wrote:

> In core/src/main/scala/org/apache/spark/h2o/H2OContext.scala
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76192289>:
>
> >
> >  /**
> >    * Create new H2OContext based on provided H2O configuration
> >    *
> >    * @param sparkContext Spark Context
> >    * @param conf H2O configuration
> >    */
> > -class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf) extends org.apache.spark.Logging
> > +class H2OContext private (@transient val sparkContext: SparkContext, @transient conf: H2OConf, sqlc: SQLContext) extends org.apache.spark.Logging
>
> I was thinking whether we need another parameter for sql context and
> change the api. We could just do SQLContext.getOrCreate(sc) inside of
> H2OContext. This returns sql context already created by spark or new one
> if none exists.
>
> What is please the intention for this ?
>
> Thanks for explanation 👍
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76192289>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK30KHoweCUoWVJdtbQJXTV4OJW3fks5qjUKKgaJpZM4Jsgi_>
> .
>